scripts/hid_configurator/configurator_core.py

```diff
@@ -639,5 +646,46 @@ def get_dfu_image_version(dfu_image):
     return ver
 
 
+class Step:
```

Comment:
this class is like struct in C, just stores data. Why doesn't it encapsulate any behavior? Take a look on a comment in ``send_continuous_led_stream``...

Reply:
This class was meant to be used this way. Used to reduce number or parameters passed to functions.